doc/nrf/protocols/zigbee/memory.rst

```diff
@@ -219,7 +219,7 @@ Values are provided in bytes (B).
       +================================================================+===============================+==============================+================================+===============================+
       | :ref:`Network coordinator <zigbee_network_coordinator_sample>` | 756                           | 2048                         | 980                            | 2048                          |
       +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
-      | :ref:`Light bulb <zigbee_light_bulb_sample>`                   | 756                           | 2048                         | 1020                           | 2048                          |
+      | :ref:`Light bulb <zigbee_light_bulb_sample>`                   | 756                           | 2048                         | 980                           | 2048                          |
```

Comment:
Should there be whitespace added to make the table look better?

Reply:
It would only affect the source table. In the built documentation, the table auto-sizes to fit the content:
![image](https://github.com/user-attachments/assets/3504e89f-c803-485a-9fcf-f9bf9e21ff35)